lib/index.js

```diff
@@ -597,10 +597,6 @@
 			const projectPath = (Project.isTitaniumApp) ? atom.project.getPaths()[0] : Project.pathForPlatform(platform);
 			args.push('--project-dir', projectPath);
 
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/index.js line...
  
  
  DangerID: danger-id-Danger;
  File: lib/index.js;
  Line: 574;
-->

- :warning: lib/index.js line 574 – 'target' is assigned a value but never used. (no-unused-vars)